sdk/ai/azure-ai-generative/azure/ai/generative/_user_agent.py

```diff
@@ -4,3 +4,4 @@
 from azure.ai.generative._version import VERSION
 
 USER_AGENT = "{}/{}".format("azure-ai-generative", VERSION)
+OPENAI_USER_AGENT = f"AzureOpenAI/Python 1.7.1 {USER_AGENT}"
```

Comment:
1.7.1 is this AzureOpenAI SDK version ?

Reply:
Yes, but gonna remove this constant. I'm going to only pass our user agent.